Nexus staging plugin failing after Java 11 deprecation

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

ci

### Describe the bug

After deprecating Java 11, the nexus-staging-maven-plugin (current version 1.6.5) is failing.



### Version

latest

### Expected behavior

Nexus plugin is able to stage the artifacts correctly

### Actual behavior

Error thrown:

`[ERROR] Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.5:deploy (injected-nexus-deploy) on project keycloak-bom-parent: Execution injected-nexus-deploy of goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.5:deploy failed: An API incompatibility was encountered while executing org.sonatype.plugins:nexus-staging-maven-plugin:1.6.5:deploy: java.lang.ExceptionInInitializerError: null`

### How to Reproduce?

Run build pipeline in https://keycloak-jenkins/view/Keycloak/job/keycloak-build-pipeline/

### Anything else?

_No response_

--- a/boms/pom.xml
+++ b/boms/pom.xml
@@ -64,7 +64,7 @@
                 <plugin>
                     <groupId>org.sonatype.plugins</groupId>
                     <artifactId>nexus-staging-maven-plugin</artifactId>
-                    <version>1.6.5</version>
+                    <version>1.6.13</version>
                     <extensions>true</extensions>
                     <configuration>
                         <serverId>jboss-releases-repository</serverId>

--- a/pom.xml
+++ b/pom.xml
@@ -198,7 +198,7 @@
         <jmeter.analysis.plugin.version>1.0.4</jmeter.analysis.plugin.version>
         <osgi.bundle.plugin.version>2.4.0</osgi.bundle.plugin.version>
         <wildfly.plugin.version>2.0.1.Final</wildfly.plugin.version>
-        <nexus.staging.plugin.version>1.6.5</nexus.staging.plugin.version>
+        <nexus.staging.plugin.version>1.6.13</nexus.staging.plugin.version>
         <frontend.plugin.version>1.12.1</frontend.plugin.version>
         <docker.maven.plugin.version>0.40.3</docker.maven.plugin.version>
         <verifier.plugin.version>1.1</verifier.plugin.version>

